Automattic/newspack-blocks
# build: remove webpack public path setting

### All Submissions:






### Changes proposed in this Pull Request:



Removes webpack's public-path setting added in e78ef06e4049046b49e34260923f20600546df89. There's no explanation in the commit message, and I can't see any reason to change the public-path, especially to Jetpack's files. 

Closes #1.

### How to test the changes in this Pull Request:

Build the plugin and verify it works as expected with and without Jetpack plugin active.

### Other information:
